### PR TITLE
ingress-node-firewall: Add E2E CI lane

### PR DESCRIPTION
**Pull request by sabinaaledort**
Deploy the operator from bundle and run E2E test.

### CODE REVIEW

**Comment by sabinaaledort** [comment]
/hold
WIP

File: ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ base_images:
     name: "4.12"
     namespace: ocp
     tag: base
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
```
Comment (msherif1234) [inline ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml:8]: is this part standary for dev-scripts who creates this ns ?

Comment (sabinaaledort) [reply ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml:8]: yes

**Review by droslean — APPROVED** [review]

**Pull request merged**